### PR TITLE
Ag inspection additions

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR adds a few fields to `AgricultureInspectionGeneric` and updates the types on that same schema.

### CODE REVIEW

**Review by BenjaminMoe — APPROVED** [review]

**Review by mkhraisha — CHANGES_REQUESTED** [review]
Many of the new schemas being pointed to are already defined in the vocabulary, is there a reason we aren't pointing directly to them?

File: docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml [hunk 1]
```diff
@@ -47,10 +47,10 @@ properties:
       $ref: ./Observation.yml
     $linkedData:
       term: observation
-      '@id': https://w3id.org/traceability#observation
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#relatedObservation
```
[the remark this answers is not in the document]
Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml:50]: I've been using the unece relatedObservation for new ag schemas (e.g. PPQ forms), but it's true that prior to that Observation references used @id https://w3id.org/traceability#observation.

I had the impression that external @id's were preferable to traceability properties so I generally look for appropriate properties in e.g. service.unece.org first, and relatedObservation I think fits pretty well. However, I'm open to suggestions here. Could you explain when you think it's appropriate to use traceability property definitions vs external definitions?

File: docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml [hunk 2]
```diff
@@ -61,7 +61,28 @@ properties:
...
       term: status
-      '@id': https://schema.org/status
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#reportedStatus
+  productQuantity:
+    title: Product Quantity
+    description: The type of package (e.g., cartons) and the number of packages present.
+    $ref: ./QuantitativeValue.yml
+    $linkedData:
+      term: productQuantity
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#Measurement
```
[the remark this answers is not in the document]
Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml:71]: This is another place where I've been defaulting to external IDs. With QuantitativeValue though there actually does seem to be a lot of precedent for using more specific external definitions.

File: docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (TallTed) [reply docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml:50]: @rhofvendahl — Please revisit your https://github.com/w3c-ccg/traceability-vocab/pull/696#discussion_r1091074112, and wrap each instance of `@id` in code fences. In this case, single backticks will work fine, i.e., ``` `@id` ```. Until you do, that GitHub user will receive notifications of every update to this PR, though they are not participating in the CCG by their own volition.

File: docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml [hunk 3]
```diff
@@ -29,14 +29,14 @@ properties:
...
-      '@id': https://schema.org/ParcelDelivery
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#transportPackage
   inspectionType:
     title: Inspection Type
     description: Type of Inspection.
     type: string
     $linkedData:
       term: inspectionType
-      '@id': https://schema.org/value
+      '@id': https://www.gs1.org/voc/certificationType
```
[the remark this answers is not in the document]
Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml:39]: I went with `https://www.gs1.org/voc/certificationType` for a few reasons:
- I've been using properties from the gs1 CertificationDetails class where possible for inspection & certificate properties, in an attempt to keep interrelated properties consistent
- While there is obviously a difference between an inspection and a certificate, inspection type and certificate type generally share the same possible values
- `value` seemed pretty vague, and I certificationType was the best I found as a more specific alternative

That said this definitely isn't a cut-and-dry case, and I'm very open to any opinions or suggestions.

File: docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml:50]: Ah, thanks Ted!

[the remark this answers is not in the document]
Comment (mkhraisha) [reply docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml:50]: relatedObservation from that link appears to be relevant to transport related observations? they have defined an observation class directly.

I'm not directly opposed to pointing from here to an external definition of an observation that makes sense. What I'm seeing is you're referring to the traceability observation yaml with its own ID, and then pointing to the uncefact related observation definition. 

if there is no difference between the two definitions of observation, then we should update all our local definitions to point to the uncefact vocabulary, if there are differences then I believe this is an error

[the remark this answers is not in the document]
Comment (OR13) [reply docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml:50]: https://w3c-ccg.github.io/traceability-vocab/#Observation is an RDF type.

https://service.unece.org/trade/uncefact/vocabulary/uncefact/#relatedObservation is an RDF Property

[the remark this answers is not in the document]
Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml:50]: @mkhraisha would it work for you to change `Observation` property references to `https://service.unece.org/trade/uncefact/vocabulary/uncefact/#relatedObservation` IFF the property is transport related?

Either way though, I think that's out of scope for this PR. Would you be OK leaving this as `uncefact/#relatedObservation` for this PR?

**Review by nissimsan — APPROVED** [review]

File: docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkhraisha) [reply docs/openapi/components/schemas/common/AgricultureInspectionGeneric.yml:50]: yes

**Review by mkhraisha — APPROVED** [review]

**Pull request merged**